### DISCUSSION TITLE
See no option to delete.

### DISCUSSION THREAD

**Discussion by borlak** [excaliburjs/Excalibur#2745] [General]
See no option to delete.

**Comment by eonarheim** [comment]
Multiplayer with physics can be tricky, depending on the game.

If your game is depending on everyone having the same physics simulation,
I'd definitely run a physics loop on the server and interpolate positions
on the client. (I'll poke around to see if we can make the excalibur
physics system on the server, but it should theoretically be possible to
run parts of excalibur both on the client and the server).

We have an example of running a matterjs loop
https://github.com/excaliburjs/sample-matterjs which can run headlessly.
